source/src/protocols/simple_moves/SampleSequenceFromProbabilities.cc

```diff
@@ -431,7 +443,7 @@ SampleSequenceFromProbabilities::calculate_position_diffs(
 		}
 
 		core::Real max_prob = 0.0;
-		for ( auto aa_and_prob : aa_probs ) {
```

Comment:
Is this why you remove the const from the function signature? You should be able to do `auto const aa_and_prob` and maintain the function signature.

Reply:
No that's not the reason, basically in the `SamplingSequenceFromProbabilities` mover I am copying the probabilities stored in the given metric once, and then pass these around sometimes modifying them. E.g. if a user has specified a task operation to restrict design, I set the probabilities of other AAs for that residue to zero. Just had a slight mixup in the previous version that is now fixed.